ChopChop/Views/Recipes/DifficultyView.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct DifficultyView: View {
+    let difficulty: Difficulty?
+
+    var body: some View {
+        if let difficulty = difficulty {
```

Comment:
I think this might be cleaner
```
let difficultyValue = difficulty?.rawValue ?? 0

ForEach(0..<difficultyValue) { _ in
    Image(systemName: "star.fill")
}

ForEach(difficultyValue..<5) { _ in
    Image(systemName: "star")
}
```